inc/sccmxml.class.php

```diff
@@ -338,7 +338,12 @@ function setNetworks() {
          foreach ($networks as $value) {
             //SCCM database store each IP format in one row, we need to split it
             //and add each IP in dedicated XML node
-            $parts = explode(",", $value['ND-IpAddress']);
+            if ($value['ND-IpAddress'] != null)
+            {
+               $parts = explode(",", $value['ND-IpAddress']);
+            } else {
+               $parts[0] = "";
+            }
```

Comment:
```suggestion
            $parts = explode(",", $value['ND-IpAddress'] ?? '');
```

Reply:
hehe, nice. 

PHP is not my confort language ;)

Reply:
done!